easybuild/easyblocks/f/fftw.py

```diff
@@ -41,7 +41,12 @@
 # Altivec (POWER), SSE (x86), NEON (ARM), FMA (x86_64)
 # asimd is CPU feature for extended NEON on AARCH64
 FFTW_CPU_FEATURE_FLAGS = FFTW_CPU_FEATURE_FLAGS_SINGLE_DOUBLE + ['altivec', 'asimd', 'neon', 'sse']
-FFTW_PRECISION_FLAGS = ['single', 'double', 'long-double', 'quad-precision']
+
+cpu_arch = get_cpu_architecture()
```

Comment:
Ah, no, that's in the sanity check...

I think it's better to leave all 4 precision 'types' to be listed in `FFTW_PRECISION_FLAGS`, and to hard set the value of `self.cfg['with_quad_prec']` to `False` on POWER, with a decent log message to go along with it (if it was set to `True` at that time)